modules/processing.py

```diff
@@ -65,8 +65,12 @@ def apply_color_correction(correction, original_image):
 def uncrop(image, dest_size, paste_loc):
     x, y, w, h = paste_loc
     base_image = Image.new('RGBA', dest_size)
+    factor_x = w // image.size[0]
+    factor_y = h // image.size[1]
     image = images.resize_image(1, image, w, h)
-    base_image.paste(image, (x, y))
+    paste_x = max(x - factor_x, 0)
+    paste_y = max(y - factor_y, 0)
+    base_image.paste(image, (paste_x, paste_y))

```

Comment:
seems to be wrong

I did a inpaing only mask at res 256x256 Denoising 0.01, 0 mask blur, 0 padding
with a mask like this
![image](https://github.com/user-attachments/assets/a0a8db3b-318d-4857-86b3-e0bb239a7616)
WITH this PR the result is visibly misaligned
![20241026-202819-009505 2123871576 0dfb](https://github.com/user-attachments/assets/0cdf57c8-f612-412e-8545-5c879f246788)

whereas without this PR seems to be perfectly aligned
![20241026-202509-740699 4031946088 f9a3](https://github.com/user-attachments/assets/d9f760fa-7296-4b0e-881f-68d8d26354dc)